src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -21,6 +21,8 @@ internal class CSharpSyntaxGenerator : SyntaxGenerator
     {
         internal override SyntaxTrivia CarriageReturnLineFeed => SyntaxFactory.CarriageReturnLineFeed;
 
+        public static readonly SyntaxGenerator Instance = new CSharpSyntaxGenerator();
```

Comment:
We should unify this with the similar pattern in CSharpFlagsEnumGenerator

- http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/CodeGeneration/CSharpFlagsEnumGenerator.cs,23
- http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/CodeGeneration/CSharpFlagsEnumGenerator.cs,44